pyface/ui/wx/window.py

```diff
@@ -19,7 +19,7 @@
 # Local imports.
 from pyface.i_window import IWindow, MWindow
 from pyface.key_pressed_event import KeyPressedEvent
-from system_metrics import SystemMetrics
+from pyface.system_metrics import SystemMetrics
```

Comment:
`system_metrics` is not in `pyface.ui.wx`, so it would have to be `from ...system_metrics import SystemMetrics`, right? It looked to me like any `pyface` imports higher up the hierarchy used the pattern `from pyface.x import x`. Maybe that's also an artifact of Python 2 style implicit local imports?

Reply:
No, it is in pyface.ui.wx: https://github.com/enthought/pyface/blob/master/pyface/ui/wx/system_metrics.py